x-pack/plugins/apm/public/components/app/TraceOverview/TraceList.tsx

```diff
@@ -76,15 +72,14 @@ const traceListColumns: ITableColumn[] = [
   }
 ];
 
-export function TraceList({ items = [], noItemsMessage, ...rest }: Props) {
-  return (
+export function TraceList({ items = [], noItemsMessage, isLoading }: Props) {
+  return isLoading ? null : (
```

Comment:
Since RRR gives us this "for free", it was easy to add and makes it a bit nicer so you don't see that brief flash of "No traces match your query" or whatever it says. Could probably do this pattern in a few other places.

Reply:
Yeah, that's a good idea.
Btw. have you watched this? :)
https://www.youtube.com/watch?v=z-6JC0_cOns

Reply:
Oh yes! Can't wait :D